packages/devtools_app/lib/src/screens/network/offline_network_data.dart

```diff
@@ -15,68 +14,49 @@ import 'network_model.dart';
 /// It is responsible for serializing and deserializing offline network data.
 class OfflineNetworkData with Serializable {
   OfflineNetworkData({
-    required this.requests,
+    required this.httpRequestData,
     this.selectedRequestId,
-    required this.currentRequests,
-    required this.socketStats,
+    required this.socketData,
   });
 
   /// Creates an instance of [OfflineNetworkData] from a JSON map.
   factory OfflineNetworkData.fromJson(Map<String, dynamic> json) {
-    final List<dynamic> requestsJson = json['requests'] ?? [];
-    final List<HttpProfileRequest>? currentReqData = json['currentRequests'];
-    final List<SocketStatistic>? socketStats = json['socketStats'];
-
-    final requests = requestsJson
-        .map(
-          (e) => DartIOHttpRequestData.fromJson(
-            e as Map<String, dynamic>,
-            null,
-            null,
-          ),
-        )
-        .toList();
+    final httpRequestData = json[OfflineDataKeys.httpRequestData.name]
+        as List<DartIOHttpRequestData>;
+    List<Socket>? socketReqData = [];
+    socketReqData = json[OfflineDataKeys.socketData.name] as List<Socket>;
```

Comment:
define in one step:
`final socketRequestData =(json[OfflineDataKeys.socketData.name] as List?)?.cast<Socket>();`

Also, prefer `socketRequestData` over `socketReqData` to avoid abbreviations in variable names